src/backend/kms/mod.rs

```diff
@@ -450,6 +451,9 @@ pub fn init_backend(
     // Create relative pointer global
     RelativePointerManagerState::new::<State>(&dh);
 
+    // TODO check if main device supports syncobj eventfd?
```

Comment:
I guess worst case we can still fail the `import_timeline`-request, right?
Is Xwayland/mesa able to handle this / fallback to implicit sync?

Reply:
I don't think there's any way to fallback once we expose the global. The protocol doesn't make `import_timeline` failable, except as a protocol error.

It looks like Mutter uses `drmSyncobjEventfd (drm_fd, 0, 0, -1, 0) != -1 || errno != ENOENT)` to check if the call is supported. So we probably want to do the same, and only create the global if it is supported.